packages/toast/src/Toast.tsx

```diff
@@ -50,13 +50,13 @@ const baseElementStyles: Partial<Record<StyledElements, string>> = {
   contentWrapper: css`
     display: flex;
     align-items: center;
-    padding: ${spacing[3]}px;
+    padding: 12px 12px 12px 18px;
```

Comment:
Since the icon has built in padding, I think the padding-left should be `16px`

Reply:
@TheSonOfThomp I see `28px`, which also happens to be incorrect in my PR lol, where do you see `32px`?

<img width="200" alt="Screen Shot 2022-01-24 at 4 22 54 PM" src="https://user-images.githubusercontent.com/5740159/150867278-c850f41e-7c5a-4a6e-b5ee-1027012808e3.png">

Reply:
The svg itself is `28px`, but the icon component itself is `32px
![Screen Shot 2022-01-27 at 11 46 35 AM](https://user-images.githubusercontent.com/2414030/151404807-2330de2f-ff3a-4f6b-86d4-962c32e2ff9f.png)
`